tools/docker/mariadb-docker/cloudstack-docker.cnf

```diff
@@ -0,0 +1,7 @@
+[mysqld]
```

Comment:
@marcaurele sorry, license please.

Reply:
I thought about it but forgot to add it before pushing. Done!